snipsskills/utils/custom_asr.py

```diff
@@ -0,0 +1,90 @@
+# -*-: coding utf-8 -*-
+""" Custom ASR installer """
+
+import getpass
+import os
+import subprocess
+
+
+from .os_helpers import execute_command, execute_root_command, get_command_output, cmd_exists, which
+
+
+class CustomASR:
+    """ CustomASR utilities. """
+
+    def __init__(self, DEBUG, asr_archive_path):
+        self.ASR_ARGUMENTS = "-v /opt/snips/asr:/usr/share/snips/asr "
+        self.separation_string = "snipsdocker/platform"
+        self.asr_archive_path = asr_archive_path
+        self.snips_command_path = self.get_snipsskills_params()
+
+    def setup(self):
+        """
+        To setup the ASR, we go through the following steps.
+        - Download the archive that contains the custom ASR model (optional)
+        - Extract the archive
+        - Move the ASR model to an appropriate location
+        - Change the `snips` script to add ASR tags
+        """
+
+        CustomASR.extract_asr_archive()
+        CustomASR.update_snips_command()
+
+    def get_snipsskills_params(self):
+        try:
```

Comment:
No need to catch an exception here, which() does the job and returns None if it fails